web/cypress/support/commands.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-undef */
+Cypress.Commands.add('login', () => {
+  const serveUrlMap = {
+    dev: 'http://139.217.190.60',
+    test: 'http://localhost:9000',
+  };
+
+  const {
+    SERVE_ENV = 'dev'
+  } = Cypress.env();
+  
```

Comment:
I think it should change the dashboard language to English first.

Reply:
> I think it should change the dashboard language to English first.

The default language of antd is English.

Reply:
but it should have a public function to do this.